FODO_Test_Lattice/Phase_Error_Propagation/README.md

```diff
@@ -0,0 +1,29 @@
+# Phase Error Propagation
+
+## Motivation 
+
+This small study aims to investigate how phase errors, 
+determined by the errors on alpha and beta of the boundary conditions,
+propagate through the lattice.
```

Comment:
not sure this is easily understandable .. 
rather saying "this small study aims to investigate how phase errors propagate through the lattice, 
using a given alpha and beta including errors as boundary condition."